cdc/kv/regionlock/region_range_lock.go

```diff
@@ -492,4 +524,5 @@ type LockedRangeValue struct {
 	RegionID     uint64
 	CheckpointTs uint64
```

Comment:
Better to reuse `LockedRange` in `LockedRangeValue `.

Reply:
`LockedRange` contains atomic things. I prefer to avoid atomic variables in the return value of `CheckLockedRanges`.

Reply:
From an implementation point of view, `resolveLock` is a relatively [low-frequency operation](https://github.com/pingcap/tiflow/pull/9163/files#diff-c82af6de18a9d7ddc2bba61db2c56944891008ec6f1cc76eb1d2b15fc68b4178R394), so the cost of reusing atomic variables is almost negligible. 

By contrast, creating multiple duplicate entities will add unnecessary complexity to the code, which is likely to be detrimental to future maintenance. At present, the concept of `LockedRange`, `LockRangeResult` and `LockedRangeValue` has been confused.

Reply:
The reason to avoid atomics in `CheckLockedRangeResult` is not about performance. Just to make it more clear.

Reply:
I have renamed this function and its return value. Please take a look again.